dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -272,31 +326,27 @@ public static void restore() throws Exception{
     }
 
     /**
-     * ContentFallback False
-     * PageFallback True
-     *
-     * Page English
-     *
-     * English -> 1 & 2
-     * Spanish -> 2 & 3
-     *
+     * Method to test: {@link HTMLPageAssetRenderedAPI#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * When: DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE is set to false and DEFAULT_PAGE_TO_DEFAULT_LANGUAGE is set to true
+     *       And the page have version just in ENG
+     *       And the page have tree content, where: content1 is just in ENG version, content2 is in ENG and ESP version, content 3 is just in ESP version
+     * Should: If the page is requests in ENG version it should be render with content1 and content2
+     *         If the page is requests in ESP version it should be render with content3 and content2 (both in ESP version)
      */
     @Test
-    public void ContentFallbackFalse_PageFallbackTrue_PageEnglish_ViewEnglishContent1And2_ViewSpanishContent2And3() throws Exception{
+    @UseDataProvider("cases")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "cases" appears 13 times in this file; the first occurrence is on line 323](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)